src/encoded/tests/data/inserts/surgery.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "patient": "KCEPT359MHZ",
+    "date": "1860-06-03",
+    "surgery_type": "Nephrectomy",
+    "nephrectomy_details": {
+      "type": "Partial",
+      "approach": "Open",
+      "robotic_assist": true,
+      "location": "UTSW"
+    },
+    "uuid": "41692393-f1f3-4797-9043-989c96a9621c"
+  },
+  {
+    "patient": "KCEPT708IJT",
+    "date": "1855-01-03",
+    "surgery_type": "Nephrectomy",
+    "nephrectomy_details": {
+      "type": "Total",
+      "approach": "Open",
+      "robotic_assist": true,
+      "location": "UTSW"
+    },
+    "uuid": "717612f1-0fb7-411b-a8b3-b045e252d098"
+  },
+  {
+    "patient": "KCEPT708IJT",
+    "date": "1860-04-03",
+    "surgery_type": "Metastectomy",
+    "nephrectomy_details": {
```

Comment:
Remove nephrectomy_details for Metastectomy and Ablation
surgery_type
location
date

Reply:
Surgery json format change, then branch broke, cannot indexing.